src/main/java/view_post/interface_adapters/ViewPostOutputData.java

```diff
@@ -0,0 +1,65 @@
+package view_post.interface_adapters;
+
+import java.util.ArrayList;
+
+public class ViewPostOutputData {
+    private final String posterEmail;
+    private final String postBody;
+    private final String postTags;
+    private final ArrayList<Integer> postReplies;
+    private final String deadline;
+    private final String creationDate;
+    private final String collaborators;
+    private final int postID;
+    private final String title;
+
+    public ViewPostOutputData(String posterEmail, String postBody, String postTags, ArrayList<Integer> postReplies,
```

Comment:
Note: we could make this smaller with a hashmap or something - super not pressing tho

Reply:
Might work on this later...